### ISSUE TITLE
Update CPU architecture note in the Proxmox installation docs

### ISSUE THREAD

**Issue by netthier** [siderolabs/talos#7402]
### Description
The docs for the installation in Proxmox currently contain this note:
![image](https://github.com/siderolabs/talos/assets/66856670/2529da0d-7705-48d8-9250-aafe9dde50d2)
As of Proxmox VE 8 (https://forum.proxmox.com/threads/proxmox-ve-8-0-released.129320/) no more workarounds are required, as the new default CPU type is now `x86-64-v2-AES`.
It might be worth adding something like "In versions prior to Proxmox VE 8, etc." to that note, as to not confuse people using newer versions.

**Comment by sergelogvinov** [comment]
better to use `host` cpu model

**Comment by reitermarkus** [comment]
@sergelogvinov, not necessarily, since using `host` prevents live migration. Unless that changed in Proxmox 8?

**Comment by netthier** [comment]
That's not entirely correct, Proxmox 8 didn't change anything, but `host` live migration generally works if the other hosts have the same CPU.
So `host` should only be avoided for heterogenous Proxmox clusters, and is fine in most other cases.
